cluster/saltbase/salt/calico/node.sls

```diff
@@ -1,30 +1,25 @@
 {% if pillar.get('network_policy_provider', '').lower() == 'calico' %}
 
-calicoctl:
-  file.managed:
-    - name: /usr/bin/calicoctl
-    - source: https://github.com/projectcalico/calico-docker/releases/download/v0.19.0/calicoctl
-    - source_hash: sha256=6db00c94619e82d878d348c4e1791f8d2f0db59075f6c8e430fefae297c54d96
-    - makedirs: True
-    - mode: 744
-
 calico-node:
-  cmd.run:
-    - name: calicoctl node
-    - unless: docker ps | grep calico-node
-    - env:
-      - ETCD_AUTHORITY: "{{ grains.api_servers }}:6666"
-      - CALICO_NETWORKING: "false"
+  file.managed:
```

Comment:
Consider also doing this in the GCI node config. I think we will be switching to that image by default for kube-up sometime.